### PR TITLE
Tons of Optimizations

### PR DESCRIPTION
**Pull request by cloudcloudskies**
Removed a lot clutter and comments
Removed lyrics generator thing
Changed wyg.mp3 with original Want You Gone
Better WantYouGone.py

### CODE REVIEW

**Comment by create-juicey-app** [comment]
Thats a lil wierd, i still want the generator to be here because i want other people to make their own creations with my project. currently seeing what you did there, but thank you a lot for supporting my code

File: WantYouGone.py
Comment (create-juicey-app) [inline WantYouGone.py:1]: hmm, seems better

File: lyrics.txt
Comment (create-juicey-app) [inline lyrics.txt:1]: what

**Comment by create-juicey-app** [comment]
reviewed most of the changes, not sure if i should merge, theres good and bad features added like the lyrics.txt changes, i dont like them, as they dont give credit to the original creators, and WYGgenerator being removed..

**Review by create-juicey-app — CHANGES_REQUESTED** [review]
need changes

**Comment by cloudcloudskies** [comment]
give me a list of changes you want

**Comment by cloudcloudskies** [comment]
I did some general optimization and did things you want

**Comment by carteeeee** [comment]
why were the code comments removed?

**Comment by create-juicey-app** [comment]
thats better, but please. dont remove code parts that seems "unessesary" like `os.environ['PYGAME_HIDE_SUPPORT_PROMPT'] = "hide"`, this WILL break the gui version.

**Comment by cloudcloudskies** [comment]
ok ima add it again

**Comment by cloudcloudskies** [comment]
> why were the code comments removed?

thats the point btw

**Comment by carteeeee** [comment]
> > why were the code comments removed?
> 
> 
> 
> thats the point btw

but then its less readable 😭

**Comment by cloudcloudskies** [comment]
yes

**Comment by create-juicey-app** [comment]
yea, im gonna review that

**Comment by cloudcloudskies** [comment]
okay

**Comment by create-juicey-app** [comment]
Currently testing the code, can refine or remake the part that explained people can make their own lyrics with this project?

File: LICENSE
Comment (create-juicey-app) [inline LICENSE:1]: Keep it!

File: README.md
Comment (create-juicey-app) [inline README.md:1]: Add back the syntax

File: WantYouGone.py
Comment (create-juicey-app) [inline WantYouGone.py:1]: Meh, minor changes, but okay

File: requirements.txt
Comment (create-juicey-app) [inline requirements.txt:1]: Really good

File: lyrics.txt
Comment (create-juicey-app) [inline lyrics.txt:1]: Meh, but ok

**Review by create-juicey-app — CHANGES_REQUESTED** [review]

**Comment by cloudcloudskies** [comment]
TODOS:
Fix GUI, disable that 01010101110 thing
Change the font to arial bold

**Comment by cloudcloudskies** [comment]
oops i broke the entire thing 💀

**Comment by create-juicey-app** [comment]
ouch
i mean my script is really bad, expect loads of shit being broken for no reason

**Comment by cloudcloudskies** [comment]
ngl yes

**Comment by cloudcloudskies** [comment]
gui version needs fix, didnt tested the normal ver

**Comment by cloudcloudskies** [comment]
u merged all prs except this

**Comment by create-juicey-app** [comment]
I'm gonna try to incorporate this on an experimental branch

**Comment by create-juicey-app** [comment]
no shit github is so fucking scary, one mistake and your main gets fucked up 😭

**Comment by create-juicey-app** [comment]
PHEW, im relieved, time to get that deserved sleep (i need to fix my sleep schedule.)

**Comment by cloudcloudskies** [comment]
lol ty for merging after years

**Comment by create-juicey-app** [comment]
> lol ty for merging after years

sorry, i spent too much time chomping on uranium

**Comment by cloudcloudskies** [comment]
npnp, do u have discord

**Comment by create-juicey-app** [comment]
> npnp, do u have discord

joober97

**Pull request merged**